addons/binding/org.openhab.binding.plugwise/src/main/java/org/openhab/binding/plugwise/internal/config/PlugwiseRelayConfig.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.plugwise.internal.config;
+
+import static com.google.common.base.CaseFormat.*;
+import static org.openhab.binding.plugwise.internal.config.PlugwiseRelayConfig.PowerStateChanging.COMMAND_SWITCHING;
+
+import org.openhab.binding.plugwise.internal.protocol.field.MACAddress;
+
+/**
+ * The {@link PlugwiseRelayConfig} class represents the configuration for a Plugwise relay device (Circle, Circle+,
+ * Stealth).
+ *
+ * @author Wouter Born - Initial contribution
+ */
+public class PlugwiseRelayConfig {
+
+    public enum PowerStateChanging {
+        COMMAND_SWITCHING,
+        ALWAYS_ON,
+        ALWAYS_OFF;
+    }
+
+    private String macAddress;
+    private String powerStateChanging = UPPER_UNDERSCORE.to(LOWER_CAMEL, COMMAND_SWITCHING.name());
+    private boolean suppliesPower = false;
+    private int measurementInterval = 60; // minutes
```

Comment:
The code has been reworked to make more use of the Java 8 Duration so exact units are more clear.

I've had a look at other bindings but most of them also don't add units to such configuration parameters. The unit of these parameters is also defined in the XML so it also is a form of duplication. It also clutters .things files when they are added. 

It would be a nice addition to ESH if it can fill Duration type properties using reflection in configuration objects! :-)